### PR TITLE
Added Incident Webhook Functionality

### PR DESCRIPTION
**Pull request by TheMajorMayhem**
With this PR I have added the ability to have your Incident reports sent to a Discord Webhook. 
This works with creating a new Incident as well as with Editing and Saving an incident. 

Steps to complete: 

- Add webhook at line 24 
- Restart Resource

This has been tested on my Dev Server, as well as on my live multiplayer server without issue. 
Preview of Webhook working: https://i.imgur.com/FpwyAvH.png

### CODE REVIEW

File: server/main.lua [hunk 1]
```diff
@@ -1888,6 +1937,25 @@ function sendToDiscord(color, name, message, footer)
 	end
 end
 
+function sendIncidentToDiscord(color, name, message, footer)
+	if ClockinWebhook == '' then
```
Comment (MonkeyWhisper) [inline server/main.lua]: IncidentWebhook

**Review by MonkeyWhisper — CHANGES_REQUESTED** [review]

File: server/main.lua (hunk 1 above)
Comment (MonkeyWhisper) [reply server/main.lua]: @xSisyphusx

Comment (TheMajorMayhem) [reply server/main.lua]: My mistake will fix this right now

Comment (TheMajorMayhem) [reply server/main.lua]: Wait - this is just the clock in webhook for the time clock - this should be correct.

Comment (MonkeyWhisper) [reply server/main.lua]: > Wait - this is just the clock in webhook for the time clock - this should be correct.

No, you're using the function sendIncidentToDiscord and sending it to ClockinWebhook, it needs to be sent to IncidentWebhook

**Review by MonkeyWhisper — CHANGES_REQUESTED** [review]

File: server/main.lua (hunk 1 above)
Comment (TheMajorMayhem) [reply server/main.lua]: @MonkeyWhisper - I have corrected this issue. My fault, for whatever reason I was having a brain fart.

**Pull request merged**